src/page-multi-facility/FacilityInputForm.tsx

```diff
@@ -25,8 +26,27 @@ import HistoricalCasesChart from "./HistoricalCasesChart";
 import LocaleInformationSection from "./LocaleInformationSection";
 import { Facility } from "./types";
 
-const ButtonSection = styled.div`
-  margin-top: 30px;
+interface ButtonSectionProps {
+  screenWidth: number;
+}
+
+const ButtonSection = styled.div<ButtonSectionProps>`
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 100px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  background-color: ${Colors.slate};
+  z-index: 1;
+  margin-left: ${({ screenWidth }) =>
+    screenWidth > 1280 ? (screenWidth - 1280) / 2 : 0}px;
+`;
+
+const PageContainerWithBottomMargin = styled(PageContainer)`
+  margin-bottom: 60px;
```

Comment:
had to compensate for new 100px fixed bar at the bottom to show the button under the age brackets